go.mod

```diff
@@ -3,9 +3,11 @@ module github.com/ONSdigital/dp-authorisation
 go 1.13
 
 require (
-	github.com/ONSdigital/dp-api-clients-go v1.9.0
+	github.com/ONSdigital/dp-api-clients-go v1.34.3
 	github.com/ONSdigital/dp-rchttp v1.0.0
-	github.com/ONSdigital/log.go v1.0.0
-	github.com/gorilla/mux v1.7.4
+	github.com/ONSdigital/go-ns v0.0.0-20200205115900-a11716f93bad // indirect
+	github.com/ONSdigital/log.go v1.0.1
```

Comment:
Is `v1` still needed?  Unless its a transitive dependency and required by one of the other dependencies we should try to do a full migration from v1 -> v2 as it would be weird to depend on both versions.

There might be a legit reason for this and it might be too big to fix now but if thats the case it would be useful to understand why its necessary. If there is reason could we  document it somewhere in case we come  back to it later please.

Reply:
ah, no probably not - I'm still getting the hand of how go get updates go.mod - the refs to v1 should be safe to remove.

Reply:
ah, should. be running `go mod tidy` as well. Fixed.